.eslintrc

```diff
@@ -1,4 +1,159 @@
 {
-  "plugins": ["@theforeman/foreman"],
-  "extends": "plugin:@theforeman/foreman/core"
+  "plugins": ["@theforeman/foreman", "spellcheck"],
+  "extends": "plugin:@theforeman/foreman/core",
+  "rules": {
+    "spellcheck/spell-checker": [
+      "warn",
+      {
+        "comments": false,
+        "identifiers": false,
+        "strings": true,
+        "lang": "en_US",
+        "ignoreRequire": false,
+        "skipWords": [
+          "2xl",
+          "altavista",
+          "ampm",
+          "auditable",
+          "bool",
+          "bootable",
+          "btns",
+          "candlepin",
+          "centos",
+          "checkbox",
+          "clearbutton",
+          "clearfix",
+          "consts",
+          "csrf",
+          "csv",
+          "datastore",
+          "datastores",
+          "datepicker",
+          "datetime",
+          "datetimepicker",
+          "dbde",
+          "debian",
+          "decrement",
+          "devs",
+          "dnk",
+          "dolphine",
+          "donut",
+          "dow",
+          "dropdown",
+          "dropdowns",
+          "duderino",
+          "ec2",
+          "erb",
+          "fieldset",
+          "fnc",
+          "formatter",
+          "fqdn",
+          "func",
+          "gettext",
+          "gluegun",
+          "glyphicon",
+          "gridster",
+          "href",
+          "internets",
+          "ip6",
+          "javascript",
+          "javascripts",
+          "jed",
+          "katello",
+          "keybind",
+          "keypress",
+          "klasses",
+          "lang",
+          "ldap",
+          "loc",
+          "locs",
+          "lsi",
+          "menuitem",
+          "monokai",
+          "mordor",
```

Comment:
do we actually have `mordor` anywhere in our code? :laughing: 
might be good to trim it to the words we actually need to ignore? from a quick look there's quite a few that shouldn't be in our code afaict. i'm not sure where this list came from, if this was auto-generated maybe it was also looking in all our dependencies (which means it will also warn us when there is a mis-spelling in them)?

Reply:
https://github.com/theforeman/foreman/blob/c1bc8cd6edc8b44b033ccc524989fb6c5eceacfe/webpack/assets/javascripts/react_app/components/ForemanModal/ForemanModalUsage.stories.js#L290

Reply:
Should we ignore storybook as well then? looks like a lot of these are just jokes/filler text from the stories

Reply:
Ahah sounds good to me :)

Reply:
And I did it manually 😭